recipes/impact-t/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set name = "impact-t" %}
+{% set version = "2.0" %}
+
+{% set build = 1000 %}
+# ensure mpi is defined (needed for conda-smithy recipe-lint)
+{% set mpi = mpi or 'nompi' %}
+
+{% if mpi == 'nompi' %}
+# prioritize nompi variant via build number
+{% set build = build + 100 %}
+{% endif %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hhslepicka/IMPACT-T/archive/v{{ version }}.tar.gz
+  sha256: a486ba796af3b7a937d168a9208c92e156c02735cb5d42b25a10fb8516e98f34
+
+build:
+  number: {{ build }}
+
+  # add build string so packages can depend on
+  # mpi or nompi variants explicitly:
+  # `pkg * mpi_mpich_*` for mpich
+  # `pkg * mpi_*` for any mpi
+  # `pkg * nompi_*` for no mpi
+
+  {% if mpi != 'nompi' %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  {% else %}
+  {% set mpi_prefix = "nompi" %}
+  {% endif %}
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}"
+
+  {% if mpi != 'nompi' %}
+
+  run_exports:
+    - {{ name }} * {{ mpi_prefix }}_*
+  {% endif %}
+
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}                    # [not win]
+    - {{ compiler('m2w64_c') }}              # [win]
+    - {{ compiler('fortran') }}              # [not win]
+    - {{ compiler('m2w64_fortran') }}        # [win]
+    - cmake
+    - {{ mpi }}  # [mpi != 'nompi']
+  host:
```

This definitely needs to be changed:
```suggestion
  host:
    - {{ mpi }}  # [mpi != 'nompi']
```

I'm not very familiar with mpi so there might be other things as well, it's worth checking if anything else needs changing to match the example in the docs: https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=mpich#message-passing-interface-mpi

---

Addressed via https://github.com/conda-forge/staged-recipes/pull/13701/commits/b3f2e1ea43915c3324f073bf35f25910e2d9bf0b